### ISSUE TITLE
Problems sections not showing characters correctly.

### ISSUE THREAD

**Issue by lintuxt** [FormidableLabs/webpack-dashboard#211]
> When executing the dashboard, I can see some HTML characters not being displayed correctly.

====================================================================

![image](https://user-images.githubusercontent.com/2812012/31489366-4099606a-aef5-11e7-9e67-ced5f17a3a22.png)

====================================================================

1) Install vue-cli
2) Generate a project with webpack
3) Include support for styl files.
4) Install webpack dashboard.
5) Duplicate code inside any styl files.
6) Run webpack dashboard.

====================================================================

```{
  "name": "n-blog",
  "description": "Blog and News Feed",
  "version": "1.0.0",
  "author": "info@netjul.com",
  "license": "MIT",
  "scripts": {
    "dev": "webpack-dashboard -- cross-env NODE_ENV=development webpack-dev-server --hot",
    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
  },
  "dependencies": {
    "vue": "^2.3.3"
  },
  "devDependencies": {
    "babel-core": "^6.0.0",
    "babel-loader": "^6.0.0",
    "babel-preset-env": "^1.5.1",
    "cross-env": "^3.0.0",
    "css-loader": "^0.25.0",
    "file-loader": "^0.9.0",
    "stylus": "^0.54.5",
    "stylus-loader": "^3.0.1",
    "vue-loader": "^12.1.0",
    "vue-template-compiler": "^2.3.3",
    "webpack": "^2.6.1",
    "webpack-dashboard": "^1.0.0-5",
    "webpack-dev-server": "^2.4.5"
  }
} 
```

```var DashboardPlugin = require('webpack-dashboard/plugin');

var path = require('path')
var webpack = require('webpack')

module.exports = {
  entry: './src/main.js',
  output: {
    path: path.resolve(__dirname, './dist'),
    publicPath: '/dist/',
    filename: 'build.js'
  },
  module: {
    rules: [
      {
        test: /\.vue$/,
        loader: 'vue-loader',
        options: {
          loaders: {
          }
          // other vue-loader options go here
        }
      },
      {
        test: /\.js$/,
        loader: 'babel-loader',
        exclude: /node_modules/
      },
      {
        test: /\.(png|jpg|gif|svg)$/,
        loader: 'file-loader',
        options: {
          name: '[name].[ext]?[hash]'
        }
      }
    ]
  },
  resolve: {
    alias: {
      'vue$': 'vue/dist/vue.esm.js'
    }
  },
  devServer: {
    historyApiFallback: true,
    noInfo: false,
    port: 4444,
    host: '0.0.0.0',
    disableHostCheck: true
  },
    plugins: [new DashboardPlugin()],
    performance: {
    hints: false
  },
  devtool: '#eval-source-map'
}

if (process.env.NODE_ENV === 'production') {
  module.exports.devtool = '#source-map'
  // http://vue-loader.vuejs.org/en/workflow/production.html
  module.exports.plugins = (module.exports.plugins || []).concat([
    new webpack.DefinePlugin({
      'process.env': {
        NODE_ENV: '"production"'
      }
    }),
    new webpack.optimize.UglifyJsPlugin({
      sourceMap: true,
      compress: {
        warnings: false
      }
    }),
    new webpack.LoaderOptionsPlugin({
      minimize: true
    })
  ])
}
```

====================================================================

- OS: OSX Sierra
- Terminal: Terminal of OSX
- 1.0.0-5
- echo $TERM = xterm-256color

**Comment by ryan-roemer** [comment]
@lintuxt -- Can you update to newly released `^1.0.0` and confirm it's still an issue?

If so, can you open a minimal repository with the error and install + reproduction steps so we have something to dig in and analyze? Thanks!

**Comment by ryan-roemer** [comment]
Closing due to inactivity / webpack-dashboard pre-2 issue. Please open a fresh issue if still happening with modern webpack-dashboard.
